modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -36,14 +35,43 @@ def submit
           render json: auto_claim
         end
 
-        def validate; end
+        def validate
+          shared_validation
+
+          ClaimsApi::AutoEstablishedClaim.new(
+            status: ClaimsApi::AutoEstablishedClaim::PENDING,
+            auth_headers:,
+            form_data: form_attributes,
+            cid: token.payload['cid'],
+            veteran_icn: target_veteran.mpi.icn
+          )
```

Comment:
interesting. we do this in v1 but it's because we call EVSS to validate it. That model has a method to convert it for the call to EVSS, but since we're not doing that additional call in v2 and relying on our validation, we could eliminate this. I don't see a need for it, but glad to hear arguments for / against. Unless we do some form of lookup / md5 to find_or_create_by in the submit, this is just junking up the DB.